## <span style="color:darkblue"> Problem Set 5: Sampling and confidence intervals

Stat140-02: Introduction to the Ideas and Applications of Statistics 

Due: Friday, March 30 @11am

**Problem set policies:** Please provide concise, clear answers for each question. Note that only writing the result of a calculation (e.g., "SD = 3.3") without explanation is not sufficient. For problems involving `R`, include the code in your solution, along with any plots.
		
Each problem set is due by 11:00am on the due date; please submit your problem set via gyrd
		
*You are encouraged to discuss problems with other students (and, of course, with the course head and the TAs), but you must write your final answer in your own words. Solutions prepared "in committee" are not acceptable. If you do collaborate with classmates on a problem, please list your collaborators on your solution.*


#### Problem 1:

Consider the standard normal distribution with mean $\mu=0$ and standard deviation $\sigma= 1$. 

a) What is the probability that an outcome $z$ is greater than 2.60?
	
b) What is the probability that $z$ is less than 1.35?
	
c) What is the probability that $z$ is between -1.70 and 3.10?
	
d) What value of $z$ cuts off the upper 15% of the standard normal distribution?
	
e) What value of $z$ marks off the lower 20% of the distribution?


#### Solution:

a) Setting `lower.tail = FALSE` instructs `R` to calculate the probability to the right of 2.6. This is equivalent to using `1 - pnorm(2.6)`. The probability that an outcome $z$ is greater than 2.6 is:

In [1]:
pnorm(2.6, lower.tail = FALSE)

[1] 0.004661188

b) The probability that $z$ is less than 1.35 is:

In [2]:
pnorm(1.35)

[1] 0.911492

c) The probability that $z$ is between -1.70 and 3.10 is:

In [3]:
pnorm(3.10) - pnorm(-1.70)

[1] 0.9544669

d) The distribution is symmetric, so for a value $z$ with area $a$ to the right, $-z$ has the same area $a$ to the left. So the value of $z$ cuts off the upper 15% of the standard normal distribution is:

In [4]:
qnorm(.15) * -1
#alternatively...
qnorm(.85)
#this also works
qnorm(.15, lower.tail=FALSE)

[1] 1.036433

[1] 1.036433

[1] 1.036433

e) The value of $z$ marks off the lower 20% of the distribution is:

In [5]:
qnorm(.20)

[1] -0.8416212

#### Problem 2:

The World Health Organization defines osteoporosis in young adults as a measured bone mineral density 2.5 or more standard deviations below the mean for young adults. Assume that bone mineral density follows a normal distribution in young adults. What percentage of young adults suffer from osteoporosis according to this criterion?



#### Solution:

Find the corresponding probability for a $z$-value of -2.5, then multiply by 100; 0.62% of young adults suffer from osteoporosis according to this criterion.

In [6]:
pnorm(-2.5) * 100

[1] 0.6209665

#### Problem 3:

Arsenic blood concentration is normally distributed with mean $\mu = 3.2$ $\mu$g/dl and standard deviation $\sigma = 1.5$ $\mu$g/dl. What range of arsenic blood concentration defines the middle 95\% of this distribution? 



#### Solution:

To find the middle 95\%, use the definition that for normally distributed data, 95\% of the data are within 1 standard deviation of the mean: $\mu \pm 1.96*\sigma$:
	
Lower bound = $3.2 - 1.96*1.5$ 

Upper bound = $3.2 + 1.96 *1.5$
	
Thus, 95% of the distribution lies in the range (0.26 $\mu$g/dl, 6.14 $\mu$g/dl).


#### Problem 4:

(Problem 1.136 in IPS, $6^{th}$ edition.) High blood cholesterol levels increase the risk of heart disease. Young women are generally less afflicted with high
cholesterol than other groups. The cholesterol levels for women aged 20 to 34 follow an approximately normal distribution with mean 185 milligrams per deciliter (mg/dl) and standard deviation 39 mg/dl.

a) Cholesterol levels above 240 mg/dl demand medical
	attention. What percent of young women have levels above 240 mg/dl?
	
b) Levels above 200 mg/dl are considered borderline high. What percent of young women have blood cholesterol between 200 and 240 mg/dl?


#### Solution:

a) First, standardize 240 mg/dl to a normal score:

$$z= \dfrac{240 -185}{39}=1.410$$


In [7]:
pnorm(1.410, lower.tail = FALSE)

[1] 0.07926984

In [8]:
#alternatively, use R to standardize and find the probability
pnorm(240, 185, 39, lower.tail = FALSE)

[1] 0.07923199

7.93% of young women have blood cholesterol levels above 240 mg/dl.

b) Standardize 200 mg/dl to a normal score:
	
$$z=\dfrac{200 -185}{39}= 0.385$$

In [9]:
pnorm(1.410) - pnorm(0.385)

#alternatively, use R to standardize and find the probability
pnorm(240, 185, 39) - pnorm(200, 185, 39)

[1] 0.2708489

[1] 0.2710292

27.1% of young women have borderline high blood cholesterol levels.

#### Problem 5:   

The General Social Survey (GSS) is a sociological survey used to collect data on demographic characteristics and attitudes of residents of the United States. In 2010, the survey collected responses from 1,154 US residents. The survey is conducted face-to-face with an in-person interview of a randomly selected sample of adults. One of the questions on the survey is "After an average workday, about how many hours do you have to relax or pursue activities that you enjoy?" A 95% confidence interval from the 2010 GSS survey for the collected answers is 3.53 to 3.83 hours.

Identify each of the following statements as true or false. Justify your answers.

  a) If the researchers wanted to report a confidence interval with a smaller margin of error based on the same sample of 1,154 Americans, the confidence interval would be larger.
  
  b) We can be 95% confident that the interval (3.53, 3.83) hours contains the mean hours that the sampled adults have for leisure time have an average workday.

  c) Suppose the researchers think that a 90% confidence level would be more appropriate. Assuming that the standard deviation remains constant, this new interval will be smaller than the 95% confidence interval.
  
  d) The confidence interval of (3.53, 3.83) contains the mean hours that U.S. adults have for leisure time after an average workday.
  
  e) The survey provides statistically significant evidence at the $\alpha = 0.05$ significance level that the mean hours U.S. adults have for leisure time after the average workday is not 3.4 hours.


#### Solution:

a) False. The confidence interval would be smaller, since the margin of error is half the width of a confidence interval (the quantity $t^\star \times \frac{s}{\sqrt{n}}$).
	
b) False. This statement would be true if the phrase "the sampled adults" were modified to "U.S. adults". Confidence intervals are statements about the population parameter $\mu$, not $\overline{x}$.
	
c) True. The new interval will be smaller, since $\alpha$ is now 0.10 instead of 0.05 and $t^\star$ decreases given that the sample size (and degrees of freedom) remains constant. It also makes sense that lower confidence corresponds with a smaller confidence interval, since the interval is now less likely to contain the population mean. 

d) False. There is a 5% chance that the interval does not contain the true population mean hours that U.S. adults have for leisure time after an average workday.

e) True. The interval corresponds to a two-sided test, with $H_0: \mu = 3.4$ hours, $H_A: \mu \neq 3.4$ hours, and $\alpha = 0.05$. Since the $\mu_0$ of 3.4 days is outside the interval, the sample provides sufficient evidence to reject the null hypothesis.

#### Problem 6:

A hospital administrator hoping to improve wait times decides to estimate the average emergency room waiting time at her hospital. She collects a simple random sample of 64 patients and determines the time between when they checked in to the ER until they were first seen by a doctor. The average time is 137.5 minutes and the standard deviation is 39 minutes. She is getting grief from her supervisor on the basis that the wait times in the ER has increased greatly from last year's average of 127 minutes. However, she claims that the increase is probably just due to chance.

a) Calculate a 95% confidence interval. Is the change in wait times statistically significantly different than last year's average of 127 at the $\alpha = 0.05$ level?
	
b) Would the conclusion in part a) change if the significance level were changed to $\alpha = 0.01$?
	
c) Is the supervisor justified in criticizing the hospital administrator regarding the change in ER wait times? How might you present an argument in favor of the administrator?



#### Solution:

a) Yes, the change in wait times is statistically significantly different than last year's average at the $\alpha = 0.05$ level; the 95% confidence interval does not contain 127 minutes.

$$\overline{x} \pm t_{df, \ 1-(\alpha/2)} \times \dfrac{s}{\sqrt{n}} $$

$$137.5 \pm  1.998 \times \dfrac{39}{\sqrt{64}} $$

$$(127.76, 147.24) $$


In [10]:
#use r as a calculator
x.bar = 137.5
s = 39
n = 64

t.star = qt(0.975, df = n - 1)

m = t.star * s/sqrt(n)

x.bar - m; x.bar + m

[1] 127.7581

[1] 147.2419

b) If the significance level were changed to $\alpha = 0.01$, then there would not be sufficient evidence to reject $H_0: \mu = 127$ minutes. We would conclude that there is not significant evidence to suggest that the wait times have changed.

$$\overline{x} \pm t_{df, \ 1-(\alpha/2)} \times \dfrac{s}{\sqrt{n}} $$

$$137.5 \pm 2.66 \times  \dfrac{39}{\sqrt{64}} $$

$$(124.55, 150.45) $$


In [11]:
#use r as a calculator
x.bar = 137.5
s = 39
n = 64

t.star = qt(0.995, df = n - 1)

m = t.star * s/sqrt(n)

t.star
x.bar - m; x.bar + m

[1] 2.656145

[1] 124.5513

[1] 150.4487

c) At the $\alpha = 0.05$ significance level, there is evidence to suggest that the average wait time in the ER has increased. However, this is not necessarily due to any particular oversight on the hospital administrator's part; other factors may be at work. For example, perhaps increased wait times are due to more patients checking into the ER this year than last year. Alternatively, a sample size of 64 is relatively small, and it is certainly possible that extreme outliers may have overly influenced the mean. It would be advisable to take a closer look at the data structure (such as through a histogram or boxplot), or repeat the test once more data has been collected.